src/scss/vendors/_vendors.scss

```diff
@@ -34,16 +35,23 @@
 				background-color: $orange-focus;
 			}
 		}
+
+		button[aria-expanded="false"] {
+			padding: 0;
+			text-align: center;
+		}
 	}
 }
 
 // UIO button style when enhance inputs is turned on
 .fl-input-enhanced #defaultContainer .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
 	display: inline-block;
 	height: rem(88);
+	max-height: rem(88);
 	padding: 0;
 
 	button {
+		font-size: 95% !important;
```

Comment:
Thanks for the nice work, @TeddTech. Although UIO buttons look really small on mobile devices but they do fit nicely in the small Galaxy Fold screen.

Reply:
Only when enhance inputs is turned off the text on the UIO buttons look small. This doesn't have to do with this line here. This line allows for the text to fit in the UIO button when enhance inputs, syllables and open dyslexic font is turned on while using the smallest screen size.

The reason why text looks small on the UIO buttons with UIO defaults settings is because the font size was changed from 16px to 12px in the revamp of visual appearance in Figma. See line 28 in `_vendors.scss` file.

Reply:
Only when enhance inputs is turned off the text on the UIO buttons look small. This doesn't have to do with this line here. This line allows for the text to fit in the UIO button when enhance inputs, syllables and open dyslexic font is turned on while using the smallest screen size.

The reason why text looks small on the UIO buttons with UIO defaults settings is because the font size was changed from 16px to 12px in the revamp of the website visual appearance in Figma. See line 28 in `_vendors.scss` file.